ci/README.md

```diff
@@ -22,12 +22,14 @@ This is a work in progress so expect things to fail or being flaky.
       cd /home/pi/RPi-Jukebox-RFID/
 * build the docker image:
       # on normal PCs:
-      docker build -t rpi-jukebox-rfid:latest -f ci/Dockerfile.amd64 .
+      docker build -t rpi-jukebox-rfid-stretch:latest -f ci/Dockerfile.stretch.amd64 .
+      docker build -t rpi-jukebox-rfid-buster:latest -f ci/Dockerfile.buster.amd64 .
       # on a raspberry pi:
-      docker build -t rpi-jukebox-rfid:latest -f ci/Dockerfile.armv7 .
+      docker build -t rpi-jukebox-rfid-stretch:latest -f ci/Dockerfile.stretch.armv7 .
+      docker build -t rpi-jukebox-rfid-buster:latest -f ci/Dockerfile.buster.armv7 .
 * get something to drink or eat
 * run the freshly built docker image and start testing. For example:
-      docker run --rm -ti rpi-jukebox-rfid:latest /bin/bash
+      docker run --rm -ti jukebox-rfid-buster:latest /bin/bash
```

Comment:
Did you miss the rpi- prefix here?

Reply:
Indeed. Thanks for catching that bit - I'll fix it shortly.